Tests/js/views/ez-subitemlistview.html

```diff
@@ -7,6 +7,40 @@
 <div class="container"></div>
 
 <script type="text/x-handlebars-template" id="subitemlistview-ez-template">
+    <td class="ez-subitem-priority-cell" id="priority-cell-42">
+        <input
+                class="ez-subitem-priority-input"
+                id="priority-42"
+                type="text"
+                readonly
+                required
+                pattern="^(\^+|-)?\d+$"
+                inputmode="numeric"
+                data-location-id="42"
+                value="{{priority}}"><div
+            class="ez-subitem-priority-icons ez-font-icon"></div><div
+            class="ez-subitem-priority-buttons"><button
+            data-priority-input="priority-42" class="ez-subitem-priority-validate ez-button ez-font-icon pure-button"></button><button
```

Comment:
this can and should be simplified to keep only what is needed by the view (so no pure-button, no ez-font-icon, no ez-button and most likely some others attribute/class can be removed as well.